packages/flutter/test/material/list_tile_test.dart

```diff
@@ -111,9 +112,12 @@ void main() {
     }
 
     void testVerticalGeometry(double expectedHeight) {
-      expect(tester.getSize(find.byType(ListTile)), new Size(800.0, expectedHeight));
-      if (hasSubtitle)
+      final Rect tileRect = tester.getRect(find.byType(ListTile));
+      expect(tileRect.size, new Size(800.0, expectedHeight));
+      if (hasSubtitle) {
         expect(top('subtitle'), greaterThanOrEqualTo(bottom('title')));
+        expect(bottom('subtitle'), lessThanOrEqualTo(tileRect.bottom));
```

Comment:
I think this should be `lessThan` for now (because min vertical padding is > 0)

Reply:
Done